webdriver-spec.html

```diff
@@ -4064,13 +4064,12 @@
 
  <li><p>Let <var>result</var> be an empty JSON <a>List</a>.
 
- <li><p>For each <var>element</var> in <var>elements returned</var>:
-
-  <ul>
-   <li><p>Append to <var>result</var>
-    the <a data-lt="JSON Serialization of an element">JSON Serialization</a>
-    of the <var>element</var>.
-  </ul>
+ <li><p>For each <var>element</var> in <var>elements returned</var>,
```

Comment:
we should then make sure that [`XPath`](https://w3c.github.io/webdriver/webdriver-spec.html#xpath) returns a `NodeList` to find. We have already done that for `link text`